spec/factories/questions.rb

```diff
@@ -1,6 +1,9 @@
 FactoryBot.define do
   factory :question do
-    text
+    text { Faker::Lorem.sentence }
```

Comment:
I used sequence for creating text of question.
https://github.com/AlinaMoskieva/MyAnswers/blob/master/spec/factories/sequences.rb#L4

You can delete it, if you do not need it anymore :)